nautobot/core/tests/test_graphql.py

```diff
@@ -526,65 +563,83 @@ def test_query_config_context(self):
 
         expected_data = {"a": 123, "b": 456, "c": 777}
 
-        response = self.client.post(
-            self.api_url,
-            data=get_device_config_context,
-            content_type="application/graphql",
-        )
-        self.assertEqual(response.status_code, status.HTTP_200_OK)
-        self.assertIsInstance(response.data["data"]["devices"], list)
-        device_names = [item["name"] for item in response.data["data"]["devices"]]
-        self.assertEqual(device_names, ["Device 1"])
+        result = self.execute_query(query)
+
+        self.assertIsInstance(result.data["devices"], list)
+        device_names = [item["name"] for item in result.data["devices"]]
+        self.assertEqual(device_names, ["Device 1", "Device 2", "Device 3"])
```

Comment:
Do we guarantee ordering of this result, or do we need a `sort` or `sorted` somewhere here?

Reply:
good point, fixed